### PR TITLE
[PT/IS] Docs fixes

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
#341

File: pytorch_toolkit/instance_segmentation/segmentoly/rcnn/losses.py [hunk 1]
```diff
@@ -127,9 +127,9 @@ def detection_loss_reg(box_deltas, classification_targets, regression_targets, c
             box_coordinates_num = 4
             boxes_num = deltas.shape[0]
             classes_num = deltas.shape[1] // box_coordinates_num
-            mask = torch.zeros((boxes_num, classes_num), dtype=torch.bool, device=device)
+            mask = torch.zeros((boxes_num, classes_num), dtype=torch.uint8, device=device)
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/instance_segmentation/segmentoly/rcnn/losses.py:130]: is it the only file where it should be done?

Comment (druzhkov-paul) [reply pytorch_toolkit/instance_segmentation/segmentoly/rcnn/losses.py:130]: Yes.

**Review by Ilya-Krylov — APPROVED** [review]

**Comment by druzhkov-paul** [comment]
@DmitriySidnev FYI

**Pull request merged**